scripts/comment_generator.py

```diff
@@ -50,25 +52,48 @@ def load_content(data_path):
         "-r", "--run_id", action="store", help="id of the run from github"
     )
     parser.add_argument(
-        "-a",
-        "--acceptance_test_report_path",
+        "-u",
+        "--path_to_urls",
         action="store",
-        help="Path to acceptance test report",
+        help="Path to urls ",
+    )
+    parser.add_argument(
+        "-x",
+        "--corrupted_sources_report_path",
+        action="store",
+        help="Path to corrupted sources report",
     )
     args = parser.parse_args()
 
     acceptance_test_report = load_content(args.acceptance_test_report_path)
+    corrupted_sources_report = load_content(args.corrupted_sources_report_path)
     urls_map = load_content(args.path_to_urls)
-    comment = "Thank you for this contribution."
-
+    comment = (
+        "Thank you for this contribution! 🍰✨🦄 \n\n"
+        "### Information about source "
+        "corruption \n\n"
+        f"{corrupted_sources_report['corruptedSourcesCount']} out of "
+        f"{corrupted_sources_report['sourceIdCount']}"
+        f" sources are corrupted."
+    )
```

Comment:
This information snippet looks really great! No need to make any additional changes right now, I just want a feature request, to provide information about how many source there were, how many broken, how many newly broken, how many corrupted, etc.

Reply:
I opened #1085 to follow up on the next steps. Feel free to comment there if you see something missing. Thank you again for all the precious feedback 🙏